### PR TITLE
Auth-web: Magic link bug fixing

### PR DESCRIPTION
**Pull request by Jxio**
*Description of changes:*
1. Name request cancel button clear the input box
2. Fix help dialog section
3. Display loading when it re-fetch affiliations

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2533-havced7i.web.app

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 1]
```diff
@@ -581,15 +581,19 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
       try {
         await this.syncOrganization(this.currentAccountSettings.id)
         await this.addOrgSettings(this.currentOrganization)
+        this.isLoading = 1 // truthy
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:584]: Why not just use a boolean?

**Review by seeker25 — APPROVED** [review]

File: auth-web/src/components/auth/manage-business/EntityManagement.vue (hunk 1 above)
Comment (Jxio) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue:584]: it used !!isLoading I think, not too sure

Comment (seeker25) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue:584]: !!isLoading 

image

**Review by seeker25 — APPROVED** [review]
Please just change it to a boolean, int makes sense if you're addressing a panel number etc, otherwise writing this.isLoading =1 // truthy  and falsy

is cleaner with 
```
this.isLoading = true
this.isLoading = false
```

**Pull request merged**